packages/fluent/package.json

```diff
@@ -32,26 +33,29 @@
 		"lodash-es": "^4.17.21"
 	},
 	"devDependencies": {
-		"@essex/jest-config": "^21.0.18",
-		"@essex/scripts": "^22.2.1",
+		"@essex/jest-config": "^21.0.19",
+		"@essex/scripts": "^23.1.4",
 		"@essex/tsconfig-base": "^1.0.2",
-		"@fluentui/react": "^8.104.2",
+		"@fluentui/react": "^8.104.5",
 		"@thematic/react": "workspace:^",
 		"@types/chroma-js": "^2.1.4",
-		"@types/d3-scale": "^4.0.2",
+		"@types/d3-scale": "^4.0.3",
 		"@types/lodash-es": "^4.17.6",
 		"@types/node": "^18.11.18",
 		"@types/react": "^18.0.26",
+		"@types/react-dom": "^18.0.10",
 		"chroma-js": "^2.4.2",
 		"jest": "^29.3.1",
-		"react": "^18.2.0"
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0"
 	},
 	"peerDependencies": {
 		"@fluentui/react": ">= 8",
 		"@thematic/react": "workspace:^",
 		"@types/node": "*",
 		"@types/react": ">= 17",
-		"react": ">= 17"
+		"react": ">= 17",
+		"react-dom": ">= 17"
```

Comment:
How do we determine if react-dom is required? We're not actually mounting anything in the libs

Reply:
It was to resolve this Yarn error with fluent: 
`@thematic/fluent@workspace:packages/fluent doesn't provide react-dom (p72e00), requested by @fluentui/react`

It's likely an optional dep, and we can resolve this warning in one of two ways:
1) Add react-dom as a dev/peer 
2) Use a packageExtensions to add `peerDependenciesMeta` to @fluentui/react